### PR TITLE
RNs-4.10.33 Added with Bug fix & security update

### PR DESCRIPTION
**Pull request by Wiharris**
RNs-4.10.33
Added with Bug fix & security update

**Version(s):**
PR applies only to enterprise-4.10

**Issue:**
https://issues.redhat.com/browse/OSDOCS-4142

**Link to docs preview:**
https://wiharris.github.io/openshift-docs/RNs-4.10.33/release_notes/ocp-4-10-release-notes.html#ocp-4-10-33

**Additional information:**
Not yet `shipped live`

### CODE REVIEW

**Review by JoeAldinger — COMMENTED** [review]
/lgtm not sure if you meant to change stuff up in 4.10.17 looks like you just deleted some extra spaces?

**Comment by kalexand-rh** [comment]
LGTM!

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -3417,4 +3417,23 @@ $ oc adm release info 4.10.32 --pullspecs
...
-To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
\ No newline at end of file
+To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-10-33"]
+=== RHBA-2022:6532 - {product-title} 4.10.33 bug fix and security update
+
+Issued: 2022-09-20
+
+{product-title} release 4.10.33, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:6532[RHBA-2022:6532] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2022:6531[RHSA-2022:6531] advisory.
```
Comment (abrennan89) [inline release_notes/ocp-4-10-release-notes.adoc:3427]: These links seem to give a 404 error for me?
Otherwise looks good!

Comment (kalexand-rh) [reply release_notes/ocp-4-10-release-notes.adoc:3427]: The links won't work until it ships.

Comment (abrennan89) [reply release_notes/ocp-4-10-release-notes.adoc:3427]: ACK, thanks!

**Pull request merged**